cmd/package.go

```diff
@@ -0,0 +1,340 @@
+		os.Exit(1)
+	}
+	err = os.MkdirAll(debDirectoryPath, 0775)
+	if err != nil {
+		fmt.Printf("hover: Failed to create deb directory %s: %v\n", debDirectoryPath, err)
+		os.Exit(1)
+	}
+
+	debDebianDirectoryPath, err := filepath.Abs(filepath.Join(debDirectoryPath, "DEBIAN"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for DEBIAN directory: %v\n", err)
+		os.Exit(1)
+	}
+	err = os.MkdirAll(debDebianDirectoryPath, 0775)
+	if err != nil {
+		fmt.Printf("hover: Failed to create DEBIAN directory %s: %v\n", debDebianDirectoryPath, err)
+		os.Exit(1)
+	}
+
+	binDirectoryPath, err := filepath.Abs(filepath.Join(debDirectoryPath, "usr", "bin"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for bin directory: %v\n", err)
+		os.Exit(1)
+	}
+	err = os.MkdirAll(binDirectoryPath, 0775)
+	if err != nil {
+		fmt.Printf("hover: Failed to create bin directory %s: %v\n", binDirectoryPath, err)
+		os.Exit(1)
+	}
+	applicationsDirectoryPath, err := filepath.Abs(filepath.Join(debDirectoryPath, "usr", "share", "applications"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for applications directory: %v\n", err)
+		os.Exit(1)
+	}
+	err = os.MkdirAll(applicationsDirectoryPath, 0775)
+	if err != nil {
+		fmt.Printf("hover: Failed to create applications directory %s: %v\n", applicationsDirectoryPath, err)
+		os.Exit(1)
+	}
+
+	controlFilePath, err := filepath.Abs(filepath.Join(debDebianDirectoryPath, "control"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for control file %s: %v\n", controlFilePath, err)
+		os.Exit(1)
+	}
+
+	controlFile, err := os.Create(controlFilePath)
+	if err != nil {
+		fmt.Printf("hover: Failed to create control file %s: %v\n", controlFilePath, err)
+		os.Exit(1)
+	}
+	controlFile.WriteString("Package: " + projectName + "\n")
+	controlFile.WriteString("Architecture: " + runtime.GOARCH + "\n")
+	controlFile.WriteString("Maintainer: @" + assertInFlutterProject().Author + "\n")
+	controlFile.WriteString("Priority: optional\n")
+	controlFile.WriteString("Version: " + assertInFlutterProject().Version + "\n")
+	controlFile.WriteString("Description: " + assertInFlutterProject().Description + "\n")
+	controlFile.WriteString("Depends: " + strings.Join(linuxPackageDependencies, ",") + "\n")
+	controlFile.Close()
+
+	binFilePath, err := filepath.Abs(filepath.Join(binDirectoryPath, "ginko"))
+	if err != nil {
+		fmt.Printf("hover: Failed to resolve absolute path for bin file %s: %v\n", binFilePath, err)
+		os.Exit(1)
+	}
+
+	binFile, err := os.Create(binFilePath)
+	if err != nil {
+		fmt.Printf("hover: Failed to create bin file %s: %v\n", controlFilePath, err)
+		os.Exit(1)
+	}
+	binFile.WriteString("#!/bin/sh\n")
+	binFile.WriteString("/usr/bin/" + projectName + "files/" + projectName + "\n")
+	binFile.Close()
+	os.Chmod(binFilePath, 0777)
+
+	splitProjectName := strings.Split(projectName, "")
```

Comment:
Use `strings.SplitN(projectName, "", 2)`, we're not interested in splitting all the characters, just the first from the rest.